photon-api/src/main/scala/com/linkedin/photon/ml/data/RandomEffectDataConfiguration.scala

```diff
@@ -48,10 +48,22 @@ protected[ml] case class RandomEffectDataConfiguration private (
     numFeaturesToSamplesRatioUpperBound: Double,
     projectorType: ProjectorType) {
 
+  /**
+   *
+   * @return
+   */
```

Comment:
I found every undocumented method and added a documentation block - my hope is that as we touch the code we'll fill them in.